### PR TITLE
Portuguese translation (pt-BR)

### PR DESCRIPTION
**Pull request by williamxsp**
Hi, im creating an portuguese translation.
Im not sure if I should finish it first or open a pull request while im translating it.

### CODE REVIEW

File: readme-pt-BR.md [hunk 1]
```diff
@@ -2,19 +2,32 @@
 
 <img src="./assets/flash.png" width="1000">
 
-[中文版](./README.zh-CN.md) [Русский](./README.ru-RU.md)
-## Introduction
+[中文版](./README.zh-CN.md) [Русский](./README.ru-RU.md) [Português](./README.pt-BR.md)
 
+## Introdução
+
+Este documento contém uma lista de boas práticas que vão nos ajudar a melhorar a performance das nossas aplicações. O "Checklist de Performance do Angular" cobre diferentes tópicos - de pré renderização no servidor e criação do pacote da nossa aplicação até performance em tempo de execu'ão e otimização do sistema de detecção de mudança feita pelo framework.
```
Comment (jlouros) [inline readme-pt-BR.md]: typo `execução`

Comment (jlouros) [inline readme-pt-BR.md]: using latest orthographic agreement `detecção` should be `deteção`
otherwise `otimização` should be `optimização`

Comment (cristianofromagio) [reply readme-pt-BR.md]: Seems like this is a brazilian portuguese translation, so both forms `detecção` and `deteção` are [accepted](https://portuguesaletra.com/acordo-ortografico/deteccao-ou-detecao-ao/). It is just that `deteção` is not used in brazilian portuguese and `detecção` is not used in european portuguese. It is the [same](https://portuguesaletra.com/acordo-ortografico/optimizar-ou-otimizar-ao/) with `otimizar` and `optimizar`.

**Comment by mgechev** [comment]
Thanks for the PR! Would love to merge it once we have the content!

File: readme-pt-BR.md (hunk 1 above)
Comment (williamxsp) [reply readme-pt-BR.md]: Thanks!

Comment (williamxsp) [reply readme-pt-BR.md]: Yes. Its a Brazilian Portuguese translation. I'll update the pull request title.

**Comment by williamxsp** [comment]
@mgechev I still need take a good read and review everything. I didn't translate some tooling sections yet but we're almost there.

**Comment by williamxsp** [comment]
Hi @mgechev I think its good to go =D

**Comment by mgechev** [comment]
Thanks for the great job and the reviews! Would you resolve the merge conflict so I can merge?

**Comment by williamxsp** [comment]
@mgechev Not sure how to resolve the merge conflicts. I recreated the readme.md at my repo.

**Comment by mgechev** [comment]
No worries, just rebased on master. Thanks again!

**Pull request merged**